### ISSUE TITLE
No photos appear after pressing show photos

### ISSUE THREAD

**Issue by shaphir** [redfish64/TinyTravelTracker#78]
Expected Behavior
After pressing the show photo, it should look like the picture below
![IMG_20180602_002733.jpg](https://ipfs.busy.org/ipfs/QmTTtt9pv3wSpXFU631dzfgwxNVJNEUbhkuiye1TsSY7CY)

Actual Behavior
After opening my application go to the dots button , and pressing show photo, but no one photo seen.

How To Reproduce
1. Open the app
2. Go to button dots and pressing show photos
3. After pressing show photos, no photos are visible
4. App: Tiny Travel Tracker version 1.1.38
5. Operating system: Oppo a57 android version 6.0.1

#### Recording Of The Bug
https://youtu.be/Mx80xPUa0U0

**Comment by NitroNils** [comment]
It has been many phones and releases since I maybe saw pictures actually loading onto the map.

**Comment by NitroNils** [comment]
Why not disable this feature if it ain't working anyway? Would trim off some bytes, wouldn't it?

**Comment by fdroiduser** [comment]
Agree. It hasn't been working since many versions (two years ago, perhaps…?)
